components/prism-sas.js

```diff
@@ -15,11 +15,11 @@
 	};
 
 	var comment = [
+		/\/\*[\s\S]+?\*\//,
```

Comment:
I just noticed this but isn't `/**/` also a valid comment?

Reply:
Agreed. Fixed in latest commit.